### DISCUSSION TITLE
Using mimirtool with loki rules

### DISCUSSION THREAD

**Discussion by Mic92** [grafana/mimir#3078] [Help and support]
I saw that https://github.com/grafana/cortex-tools is deprecated, however there still seems no replacement for `cortextool rules lint --backend=loki`. Am I missing something?

**Comment by pracucci** [comment]
👋  https://github.com/grafana/cortex-tools is not officially deprecate because it's still used by Loki. `mimirtool` has no plans to add Loki support, but Loki team is condering building their own `lokitool` (as far as I know no decision has been taken yet). In the meanwhile, you should keep using cortex-tools with Loki.

**Reply by Ulexus** [reply]
Is this still true?  cortex-tools indicates that we should come here, and there is no `lokitool`, so far as I can find.

**Reply by Ulexus** [reply]
For instance, this doc indicates that mimirtool is to be used to load Loki alert rules, but nothing explains how, and when I try, mimirtool doesn't like the rule format:

https://grafana.com/docs/grafana-cloud/legacy-alerting/alerts-rules/

**Reply by snk-actian** [reply]
@Ulexus Still no `lokitools` (as far as I could search) and no Loki support in `mimirtool`, I still need to use `cortextool` with `--backend loki` option

**Reply by Ulexus** [reply]
Yeah, that's also what support just told me.  Thanks

**Reply by dehimb** [reply]
Hello

Is there any updates on this?
